### ISSUE TITLE
Querying on boolean attributes doesn't work when they are false.

### ISSUE THREAD

**Issue by Ramblurr** [funkensturm/ember-local-storage#57]
The LS Adapter doesn't support querying on boolean model attributes when the values are false:

From [adapter.js:217](https://github.com/funkensturm/ember-local-storage/blob/master/addon/adapters/adapter.js#L217)
```js
        // Attributes
        if (key === 'id' || key === 'type') {
          recordValue = data[key];
        } else {
          key = serializer.keyForAttribute(key);
          recordValue = data.attributes ? data.attributes[key] : null;
        }

        if (recordValue) { // <----------------- this is the problem line
          return this._matches(recordValue, queryValue);
        }
```

If the model has an attribute  like `isAwesome: DS.attr('boolean', {defaultValue: false}),`

Then a query like `store.query('my-model', { filter: {isAwesome: false } })` returns 0 results.

**Comment by fsmanuel** [comment]
It's a bit odd but can you try:
`store.query('my-model', { filter: {isAwesome: 'false' } })`

I'm not 100% how the boolean is saved in localStorage. I seem to remember that I had problems with numbers as well...

We should probably use `Ember.isEmpty` to fix it.

**Comment by Ramblurr** [comment]
I've tried treating the boolean as a string, but it doesn't work. The [`Boolean` transform](https://github.com/emberjs/data/blob/v2.3.0/addon/-private/transforms/boolean.js#L40) serializes as a Boolean, which will get converted to a regular boolean when placed in LS.

I had problems with integers too, and treating them as strings was my workaroun.

But yea, Ember.isEmpty is the answer here imho.

**Comment by fsmanuel** [comment]
I'll fix it tomorrow!

**Comment by fsmanuel** [comment]
@Ramblurr I released `0.1.4`
